test/examples/boostrap.js

```diff
@@ -0,0 +1,179 @@
+/* global process, Promise, before, after */
+// eslint-disable-next-line import/no-extraneous-dependencies
+const puppeteer = require('puppeteer');
+const { fork } = require('child_process');
+const net = require('net');
+const fs = require('fs');
+const os = require('os');
+const path = require('path');
+
+// We could run 'npm start' to serve itowns for the tests,
+// but it's slow to start (so tests might fail on timeouts).
+// Since the 'test-examples' target depends on the 'run' target,
+// we instead run the simplest http server.
+// It needs to be forked, so it's written to a temp file before
+// running.
+const simplestHttpServer = `
```

Comment:
I see no problem with moving this to its own file, like in `test/examples/server.js` maybe ?